invenio_app/ext.py

```diff
@@ -47,6 +51,27 @@ def init_app(self, app, **kwargs):
             self.talisman = Talisman(
                 app, **app.config.get('APP_DEFAULT_SECURE_HEADERS', {})
             )
+        # Enable PING view
+        if app.config['APP_HEALTH_BLUEPRINT_ENABLED']:
```

Comment:
It's probably better to move this to a ``create_blueprint_from_app(app)`` function similar to in [Invenio-Records-REST](https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/views.py#L116) and [here](https://github.com/inveniosoftware/invenio-records-rest/blob/master/setup.py#L124). This way the blueprint gets registered during the blueprint phase and ensures that all extensions have been loaded first.
Also, this would allow another extension to set the ``APP_HEALTH_BLUEPRINT`` variable.